app/src/main/res/values/strings.xml

```diff
@@ -27,6 +27,14 @@
     <string name="downloadImage">Download Image</string>
     <string name="imageOptions">Image Options</string>
     <string name="permissionRequiredToDownload">Downloading requires storage permission</string>
+    <string name="find_in_page">Find in page</string>
+
+    <!-- Find in page -->
+    <string name="nextSearchTermDescription">Find next</string>
+    <string name="previousSearchTermDescription">Find previous</string>
+    <string name="closeFindInPageButtonDescription">Close find in page view</string>
```

Comment:
This is marked as unused

Reply:
Fixed (something _should_ have been using it)